integr8ly/tutorial-web-app
# Global wt resources

## Motivation

Add a new annotation: `[type=walkthroughResources]`. Also adds some tests for the custom walkthrough parsing.

## What

Adds a new annotation that allows users to specify 'global' resources that will be displayed on the right hand side of the tutorial overview as well as on the individual tasks. 

## Why

There was no way to specify walkthrough-level resources yet.


## Verification Steps

Check out this branch, start the webapp locally and open the demo walkthrough.

1. Make sure that on the tutorial overview page you can see `A walkthrough resource`
1. Open the first task. You should also see `A walkthrough resource`

## Checklist:


## Progress
